Source/WebKit/Platform/cocoa/SharedMemoryCocoa.cpp

```diff
@@ -95,17 +96,23 @@ SharedMemory::Handle::~Handle()
     clear();
 }
 
-void SharedMemory::Handle::takeOwnershipOfMemory(MemoryLedger memoryLedger) const
+void SharedMemory::Handle::takeOwnershipOfMemory(const ProcessIdentity& processIdentity, MemoryLedger memoryLedger) const
```

Comment:
I think we should introduce a new function called setOwnershipOfMemory(const ProcessIdentity&, MemoryLedger) instead.

Having something called "takeOwnershipOfMemory" but passing the identity of another process in parameter is super confusing.

Reply:
The new function SharedMemory::setOwnershipOfMemory() was added.